backend/app.py

```diff
@@ -1,5 +1,6 @@
-from flask import Flask, send_from_directory, jsonify
+import epub_reader
```

Comment:
We won't be converting the book on server run, the conversion will be done and then stored separately so you can just omit this